docs/docs/guides/labs/dg/configuring-dg.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Configuring dg'
+sidebar_position: 500
+---
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+`dg` can be configured from both configuration files and the command line.
+There are three kinds of settings:
+
+- Application-level settings configure the `dg` application as a whole. They can be set
+  in configuration files or on the command line, where they are listed as
+  "global options" in the `dg --help` text.
+- Project-level settings configure a `dg` project. They can only be
+  set in the configuration file for a project.
+- Workspace-level settings configure a `dg` workspace. They can only
```

Comment:
Worth inline link to the "multiple projects guide". 

(See [guide] for more details about workspaces)